news/news.rst

```diff
@@ -1,10 +1,14 @@
-**Added:** 
+**Added:**
```

Comment:
This file shouldn't be named news, please go with something more descriptive for this branch